### PR TITLE
change title

### PR DESCRIPTION
**Pull request by iamnody**
Document ---> git-web

### CODE REVIEW

**Review by thmsnody — COMMENTED** [review]
test
edit?

**Review by cptegn — CHANGES_REQUESTED** [review]
remove dash
git web
not
git-web

**Comment by cptegn** [comment]
> test edit?

Don't delete or remove

>

**Comment by cptegn** [comment]
replay

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>git-web</title>
```
Comment (cptegn) [inline index.html:7]: no dash is better
git web

Comment (cptegn) [reply index.html:7]: > no dash is better git web

test

Comment (iamnody) [reply index.html:7]: ??

Comment (iamnody) [inline index.html:7]: dash

File: index.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
```
Comment (iamnody) [inline index.html:7]: first

File: index.html (hunk 1 above)
Comment (cptegn) [inline index.html:7]: new

Comment (cptegn) [reply index.html:7]: dash new

**Review by iamnody — COMMENTED** [review]
sumit test

**Review by thmsnody — APPROVED** [review]

**Comment by iamnody** [comment]
merge

**Review by cptegn — APPROVED** [review]
approve

**Comment by thmsnody** [comment]
> left a comment



> sumit test

2

**Comment by thmsnody** [comment]
> sumit test

222

**Comment by cptegn** [comment]
> > sumit test
> 
> 222
333

**Review by thmsnody — COMMENTED** [review]
aaa

**Review by cptegn — APPROVED** [review]
www

**Pull request merged**